HUD/SetMinimapComponent.md

```diff
@@ -9,14 +9,25 @@ Any SET_MINIMAP_COMPONENT(int componentID, BOOL toggle, int hudColor);
 ```
 
 This native is used to colorize certain map components like the army base at the top of the map.
+The provided color will be mixed with the existing component color. For example using blue (HudColor: 9) on the yellow lift-off markers will result in green.
 
-An incomplete list of components ID:
+**A list of components IDs:**
 
-0: Los Santos' air port yellow lift-off markers.
-1: Sandy Shore's air port yellow lift-off markers.
-2: Trevor's air port yellow lift-off markers.
-6: Vespucci Beach lifeguard building.
-15: Army base.
+* 0: Los Santos' air port yellow lift-off markers. 
+* 1: Sandy Shore's air port yellow lift-off markers. 
+* 2: Trevor's air port yellow lift-off markers. 
+* 3: ? 
+* 4: Unused 
+* 5: Unused
+* 6: Vespucci Beach lifeguard building. 
+* 7: "Beam Me Up" Upper area (Alien hill in Sandy Shores). 
+* 8: Paleto Bay Fire Department Tower. 
+* 10: Paleto Bay Aerial Tramway station 
+* 11: Electricity stuff at the Observatory. 
+* 12: Roof of some building in Strawberry. 
+* 13: Richman Playboy Mansion pool cave. 
+* 14: "Beam Me Up" Circles (Alien hill in Sandy Shores). 
+* 15: Army base. 
 
 [List of hud colors](https://pastebin.com/d9aHPbXN)
```

Comment:
The pastebin doesn't exist anymore. There's a list of colors here:
https://docs.fivem.net/docs/game-references/hud-colors/

I fixed this in the past in the following PR:
https://github.com/citizenfx/natives/pull/838/files#diff-9da3e7fd7e8fa3c3796a2cba295ad08f1c9e71eefda07dfbb9d6947999ca8cc2